src/main/java/org/myrobotlab/service/Runtime.java

```diff
@@ -598,9 +598,16 @@ static public boolean setAllVirtual(boolean b) {
   /**
    * start
    * @param b
+   * @throws IOException 

```

Comment:
Under what circumstances is it thrown? Also the javadoc should explain what auto starting means, especially what file it will be using